plugin.video.sgtv/resources/search.xml

```diff
@@ -0,0 +1,2 @@
+(lp0
```

Comment:
You can define it as `__profile__ = xbmc.translatePath(__addon__.getAddonInfo('profile')).decode('utf-8')` and then just write to a `search.xml` file in that folder. The file does not exist when you ship the addon but it will be created once the user starts to use the addon. Note that this change is not mandatory but it is generally a good principle. If you ship the search.xml within the addon, it will be re-written every update. So, taking into account this file is to store the users' search history, it will be lost every time you issue an update.